staging/src/k8s.io/api/flowcontrol/v1alpha1/types.go

```diff
@@ -277,13 +277,7 @@ type NonResourcePolicyRule struct {
 	// +listType=set
 	// Required.
 	Verbs []string `json:"verbs" protobuf:"bytes,1,rep,name=verbs"`
-	// `nonResourceURLs` is a set of url prefixes that a user should have access to and may not be empty.
-	// For example:
-	//   - "/healthz" is legal
```

Comment:
Oh, we didn't implement it this way?

Reply:
No, we never got around to implementing that level of sophistication.
If you think it is needed for this release then I can solve this inconsistency the other way.

Reply:
Note also that, although we started out by copying from rbac, we evolved to a much more finicky definition for NonResourceURLs than rbac has.  See the implementation for rbac at https://github.com/kubernetes/kubernetes/blob/aeb88b6ecd898bbb788a4629d5e0163172c6c85f/pkg/apis/rbac/v1/evaluation_helpers.go#L100-L110

Reply:
See #88794

Reply:
Thanks, I like that one better, will comment there.

Reply:
Yeah, I think we should fix our validation. RBAC did this wrong IMO, that's why we clarified the docs here.